### ISSUE TITLE
edit_features method does not accept list of Features when trying to add Features to a Feature Service

### ISSUE THREAD

**Issue by apulverizer** [Esri/arcgis-python-api#83]
In the documentation/comments it says:

> adds - The array of features to be added.
> updates - The array of features to be updateded.
> deletes - string of OIDs to remove from service

Yet the function only accepts a list/array of dictionaries or a list/array of PropertyMaps when trying to add items. When trying to update Features, it handles this correctly.

layer.py line 533
```python
if isinstance(adds, FeatureSet):
    params['adds'] = json.dumps([f.as_dict for f in adds.features],
                                default=_date_handler)
elif len(adds) > 0:
    if isinstance(adds[0], dict):
        params['adds'] = json.dumps([f for f in adds],
                                    default=_date_handler)
    elif isinstance(adds[0], PropertyMap):
        params['adds'] = json.dumps([dict(f) for f in adds],
                                    default=_date_handler)
    else:
        print('pass in features as dict or PropertyMap')
if isinstance(updates, FeatureSet):
    params['updates'] = json.dumps([f.as_dict for f in updates.features],
                                   default=_date_handler)
elif len(updates) > 0:
    if isinstance(updates[0], dict):
        params['updates'] = json.dumps([f for f in updates],
                                       default=_date_handler)
    elif isinstance(updates[0], PropertyMap):
        params['updates'] = json.dumps([dict(f) for f in updates],
                                       default=_date_handler)
    elif isinstance(updates[0], Feature):
        params['updates'] = json.dumps([f.as_dict for f in updates],
                                       default=_date_handler)
    else:
        print('pass in features as dict or PropertyMap')
```

I expected this function to accept a list of arcgis.features.Feature and process add those.

The workaround is to create a arcgis.features.FeatureSet from the original set of features like:

```python
my_feature_layer.edit_features(adds=arcgis.features.FeatureSet(list_of_features_to_add))
```

**Comment by rohitgeo** [comment]
Thank you for reporting this. We've fixed this internally and the fix will be included in the next update.

**Comment by rohitgeo** [comment]
The fix is included in v1.2.1 that was released last week.
